### ISSUE TITLE
[Accessibility] The element's name is missing or incorrect

### ISSUE THREAD

**Issue by michalina-graczyk** [mirumee/saleor-storefront#833]
### What I'm trying to achieve is to have element's names

Summary | Selector
-- | --
The element's accessible name is missing or is not appropriate for the element's purpose. | ["body > div#root:nth-of-type(1) > header:nth-of-type(1) > nav#header.main-menu:nth-of-type(1) > div.main-menu__right:nth-of-type(3) > ul:nth-of-type(1) > div.menu-dropdown:nth-of-type(1) > li.main-menu__icon.main-menu__user--active:nth-of-type(1) > div:nth-of-type(1) > div:nth-of-type(1) > svg.injected-svg:nth-of-type(1)"]
The element's accessible name is missing or is not appropriate for the element's purpose. | ["body > div#root:nth-of-type(1) > header:nth-of-type(1) > nav#header.main-menu:nth-of-type(1) > div.main-menu__right:nth-of-type(3) > ul:nth-of-type(1) > li.main-menu__icon.main-menu__cart:nth-of-type(1)"]
The element's accessible name is missing or is not appropriate for the element's purpose. | ["body > div#root:nth-of-type(1) > header:nth-of-type(1) > nav#header.main-menu:nth-of-type(1) > div.main-menu__right:nth-of-type(3) > ul:nth-of-type(1) > div.menu-dropdown:nth-of-type(1) > div.menu-dropdown__body:nth-of-type(1)"]

**System information**
Operating system: MacOS
Browser: Chrove v.84
